### PR TITLE
2021Q4 portmgr entry

### PR DESCRIPTION
**Pull request by rene0**

### CODE REVIEW

File: 2021q4/portmgr.adoc [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+There was also a change in portmgr: adamw@ stepped down after five years of service and tcberner@ is now a full member of portmgr@.
+
+Three new USES were introduced:
+* magick to handle dependencies on ImageMagick
+* nodejs to provide support for NodeJS (with a new default version NODEJS=lts)
+* trigger to handle pkg triggers using the TRIGGERS variable
+The default version of PGSQL switched to 13.
+Furthermore, INSTALLS_ICONS has been replaced by a trigger on gtk-update-icon-cache, the macro is no longer functional.
+
+As always, there were some updates to "big" packages: pkg was updated to 1.17.5, Chromium to 94.0.4606.81_3, Firefox to 95.0.2_1,2, and Ruby 3.1.0 is now available for use by users and other ports.
```
Comment (pauamma) [inline 2021q4/portmgr.adoc:31]: I saw a "3.11 python update too!" reply on ports@, in case you you want to add that.

**Review by pauamma — CHANGES_REQUESTED** [review]
Can you resubmit with just this quarter's changes to portmgr-template.adoc and 2021q4/portmgr.adoc (with or without python 3.11)? As it is, the presence of a bunch of q3 commits prevents one-click merging of this pull request.

File: 2021q4/portmgr.adoc (hunk 1 above)
Comment (rene0) [reply 2021q4/portmgr.adoc:31]: Ah, I can add Python 3.11.

**Comment by rene0** [comment]
Hmm, something must be out-of-sync with my fork? The extraneous commits should not do anything though, judging from the diff?

**Review by pauamma — APPROVED** [review]

**Comment by rene0** [comment]
I will refork this for a clean pull request.

**Pull request closed without merging**